libdnf5/rpm/solv/goal_private.cpp

```diff
@@ -128,14 +127,19 @@ bool can_depend_on(Pool * pool, Solvable * sa, Id b) {
     return false;
 }
 
-static void same_name_subqueue(libdnf5::solv::RpmPool & pool, Queue * in, Queue * out) {
+
+/// @brief It takes an `in` Queue which contains packages sorted by name (see `installonly_cmp`).
+///        It selects the first name and upon return the `out` Queue is populated with packages
```

Comment:
May be I do not understand it correctly. I don't see that it states that it moves all packages (Ids) with the name that is returned from `in` to `out` Queue. Or did I overlooked something?

Reply:
Hopefully its better now.